static/app/views/starfish/utils/sqlish/SQLishFormatter.tsx

```diff
@@ -47,7 +47,10 @@ export class SQLishFormatter {
     try {
       tokens = this.parser.parse(sql);
     } catch (error) {
-      Sentry.captureException(error);
+      Sentry.withScope(scope => {
+        scope.setFingerprint(['sqlish-parse-error']);
+        Sentry.captureException(error);
```

Comment:
Might be nice if we could log the `sql` value here as well? The errors right now don't look very useful